src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -2071,38 +2378,23 @@ open_common(int (*real_open)(const char *pathname, int oflags, ...), const char
 org_func:
 	FREE(parent_dir);
 	if (two_args)
-		return real_open(pathname, oflags);
+		fd_kernel = real_open(pathname, oflags);
 	else
-		return real_open(pathname, oflags, mode);
+		fd_kernel = real_open(pathname, oflags, mode);
+	return fd_kernel;
+
```

Comment:
This part of the change isn't required.

Reply:
You are right. Reverted unneeded changes here. Thank you!